gpcontrib/arenadata_toolkit/arenadata_toolkit--1.3--1.4.sql

```diff
@@ -0,0 +1,399 @@
+/* gpcontrib/arenadata_toolkit/arenadata_toolkit--1.3--1.4.sql */
+
+DO $$
+BEGIN
+	IF EXISTS (SELECT 1
+			   FROM pg_tables
+			   WHERE schemaname = 'arenadata_toolkit' AND
+			         tablename = 'db_files_current')
+	THEN
+		-- Use "IF" into another "IF" because in other case "<table_name>::regclass"
+		-- leads to error if the such table does not exist.
+		IF NOT EXISTS (SELECT 1
+					   FROM pg_attribute
+					   WHERE attname = 'tablespace_location' AND
+					         NOT attisdropped AND
```

Comment:
What's the motivation behind using `NOT attisdropped `? If attribute is dropped we suppose that it exists anyway and there is no need to add this column? Is it expected behaviour?